ui/admin/app/components/form/role/manage-scopes/index.hbs

```diff
@@ -63,10 +68,15 @@
   {{/if}}
 
   {{! TODO: Add correct route string once route is created}}
+  {{! TODO: Add check-circle icon if custom scopes were added}}
   <Hds::Button
     @text={{t 'resources.role.scope.actions.add-custom-scopes.text'}}
     @color='secondary'
-    @route='scopes.scope.roles.role'
+    @route={{if
+      @model.scope.isGlobal
+      'scopes.scope.roles.role.manage-scopes.manage-custom-scopes'
+      'scopes.scope.roles.role.manage-scopes'
+    }}
```

Comment:
Should we be able to go to the mange custom scopes page for a role on an org? Or is this for another PR?

https://github.com/hashicorp/boundary-ui/assets/5783847/0cc9990b-fd1e-4760-9533-2b83f64de6f6

Reply:
That will come in a different PR since the page looks different and lists only projects for selection.